perceval/archive.py

```diff
@@ -342,3 +343,123 @@ def _count_table_rows(self, table_name):
             cursor.close()
 
         return row[0]
```

Comment:
LGTM the change from item_category to category. It is something that can bite us, but category is not used in general now, so it is the moment to do this change.